narwhal/types/src/consensus.rs

```diff
@@ -150,6 +183,28 @@ impl ConsensusStore {
         s
     }
 
+    /// Returns thet latest subdag committed. If none is committed yet, then
+    /// None is returned instead.
+    pub fn get_latest_sub_dag(&self) -> Option<CommittedSubDagShell> {
+        self.committed_sub_dags_by_index
+            .iter()
+            .skip_to_last()
+            .next()
+            .map(|(_, subdag)| subdag)
+    }
+
+    /// Returns the subdag by the specified index. If found Some is returned with the result,
+    /// otherwise None is returned instead.
+    pub fn get_sub_dag_by_index(
```

Comment:
it appears this function is unused

Reply:
Good point! I've added it as util method, but is not needed anymore